Application/Tests/run_unix.bash

```
@@ -184,112 +215,120 @@ function check_frame_count_with_pvinfo() {
-echo "Running TGrabs... ${CMD}"
-if ! { ${CMD} 2>&1; } > "${PWD}/tgrabs.log"; then
-    cat "${PWD}/tgrabs.log"
-    echo "TGrabs could not be executed."
+rm -f "${WPWD}/corrected/test.settings"
+# Conversion writes to the un-prefixed output dir; keep the prefix and the
+# scanned data dir in lock-step via CONVERT_PREFIX.
+CONVERT_PREFIX=""
+CONVERT_DATA_DIR="$(data_dir_for_prefix "${CONVERT_PREFIX}")"
+CONVERT_P_ARG=""
+[[ -n "${CONVERT_PREFIX}" ]] && CONVERT_P_ARG="-p ${CONVERT_PREFIX} "
+CMD="${CONVERT} -d "${WPWD}" -i \"${TEST_FRAMES_DIR}/frame_%3d.jpg\" -o test -s \"${WPWD}/test.settings\" -auto_quit -nowindow -task convert -detect_type background_subtraction ${CONVERT_P_ARG}-history_matching_log history_matching_convert.html"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)